src/client/driver/status.js

```diff
@@ -4,6 +4,7 @@ export default class DriverStatus extends Assignable {
     constructor (obj) {
         super(obj);
 
+        this.id              = Date.now();
```

Comment:
AFAIK it is possible to get response from local machine within 1ms. So, yes, we need better precision